models/db/db.js

```diff
@@ -5,7 +5,10 @@ mongoose.Promise = global.Promise;
 const config = require('./config');
```

Comment:
we don't want config files in the repo? Can't see it in .gitignore.

Reply:
This config file is just abstracting a environment variable, there is no config in it. @nikrb but i've wanted to get rid of that abstraction bc there is no point @vivekimsit